AddressBook_/AddressBook_/DetailViewController.swift

```diff
@@ -0,0 +1,368 @@
+        yesButton.isSelected = !noButton.isSelected
+        
+        yesButton.setBackgroundColor(button: yesButton, isSelected: yesButton.isSelected )
+        noButton.setBackgroundColor(button: noButton, isSelected: noButton.isSelected )
+    }
+    
+    // ********************************
+    // MARK: - initialSetup
+    // ********************************
+    func initialSetup() {
+        // 기본 배경 색깔 설정
+        view.backgroundColor = .white
+        
+        // 기본 텍스트 설정
+        backButton.setTitle("back", for: .normal)
+        saveButton.setTitle("save", for: .normal)
+        addImageButton.setTitle("사진 등록", for: .normal)
+        yesButton.setTitle("YES", for: .normal)
+        noButton.setTitle("NO", for: .normal)
+        nameLabel.text = "Name"
+        phoneLabel.text = "Phone"
+        favoriteLabel.text = "Favorite"
+        memoLabel.text = "memo"
+        
+        // 뒤로가기 버튼
+        backButton.titleLabel?.font = UIFont.systemFont(ofSize: 15, weight: .bold)
+        backButton.setTitleColor(.black, for: .normal)
+        backButton.setTitleColor(.systemBlue, for: .highlighted)
+        backButton.layer.borderWidth = 1
+        backButton.clipsToBounds = true
+        backButton.layer.cornerRadius = 5
+        
+        // 저장 버튼
+        saveButton.titleLabel?.font = UIFont.systemFont(ofSize: 15, weight: .bold)
+        saveButton.setTitleColor(.black, for: .normal)
+        saveButton.setTitleColor(.systemBlue, for: .highlighted)
+        saveButton.layer.borderWidth = 1
+        saveButton.clipsToBounds = true
+        saveButton.layer.cornerRadius = 5
+        
+        // 이미지 뷰
+        let testImage = UIImage(systemName: "pencil")
+        memberImageView.image = testImage
+        memberImageView.backgroundColor = .white
+        memberImageView.layer.borderWidth = 1
+        memberImageView.clipsToBounds = true
+        memberImageView.layer.cornerRadius = (view.frame.width / 2.5) / 2
+        
+        // 이미지 저장 버튼
+        addImageButton.titleLabel?.font = UIFont.systemFont(ofSize: 12, weight: .bold)
+        addImageButton.setTitleColor(.black, for: .normal)
+        addImageButton.setTitleColor(.systemBlue, for: .highlighted)
+        addImageButton.layer.borderWidth = 1
+        addImageButton.layer.cornerRadius = 5
+        addImageButton.clipsToBounds = true
+        
+        // 이름 레이블
+        nameLabel.textColor = .black
+        nameLabel.font = UIFont.systemFont(ofSize: 15, weight: .bold)
+        nameLabel.textAlignment = .left
+        
+        // 번호 레이블
+        phoneLabel.textColor = .black
+        phoneLabel.font = UIFont.systemFont(ofSize: 15, weight: .bold)
+        phoneLabel.textAlignment = .left
+        
+        // 이름 텍스트 필드
+        nameTextField.borderStyle = .line
+        nameTextField.textColor = .black
+        nameTextField.font = UIFont.systemFont(ofSize: 15, weight: .light)
+        nameTextField.leftView  = UIView(frame: CGRect(x: 0.0, y: 0.0, width: 8.0, height: 0))
```

Comment:
```suggestion
        nameTextField.leftView  = UIView(frame: CGRect(x: 0, y: 0, width: 8, height: 0))
```

소수점 자리가 `0`으로 떨어지는 경우는 `.0`을 생략하여 표기합니다.